lib/game_loop.rb

```diff
@@ -27,7 +27,11 @@ def core_game
         user_choice = @questions.validate_user_input("PICK A SPACE #{@game_logic.current_player()}", valid_options)
         @game_logic.take_turn(user_choice)
       else
-        @computer_logic.cpu_turn
+        if @game_config.cpu_difficulty == :easy
+          @computer_logic.cpu_turn_easy
+        else
+          @computer_logic.cpu_turn_hard
+        end
```

Comment:
There is no reason that the game loop should be modified for this change.

Reply:
Got it! It was a doozy adjusting test specs to pass but figured it out